openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomControls.component.tsx

```diff
@@ -192,7 +191,7 @@ const CustomControls: FC<ControlProps> = ({
           disabled={!hasEditAccess}
           title={hasEditAccess ? 'Edit Lineage' : NO_PERMISSION_FOR_ACTION}
           onClick={onEditLinageClick}>
-          {getLoadingStatus()}
+          {LoadingStatus(editIcon, loading, status)}
```

Comment:
Sorry for not catching it earlier but function name should be more descriptive & naming should be in camelCase